src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+import static java.lang.Math.PI;
+
+import java.util.Random;
+
+public class Circle extends Figure {
+    private static final int MAX_NUMBER = 100;
+    private int radius;
+    private final Random random = new Random();
+    private final ColorSupplier color = new ColorSupplier();

```

Comment:
We need only `radius` field
```suggestion
    private int radius;
```